sunpy/data/_sample.py

```diff
@@ -51,52 +65,58 @@
     "NORH_LIGHTCURVE": ("tca110810", "")
 }
 
-sample_files = {}
-for key in _files:
-    sample_files[key] = os.path.abspath(os.path.join(sampledata_dir, _files[key][0]))
+
+free_required=145227776
+if disk_usage('/').free > free_required :
+
+	sample_files = {}
+	for key in _files:
+    		sample_files[key] = os.path.abspath(os.path.join(sampledata_dir, _files[key][0]))
```

Comment:
It seems you've inserted tabs in this line. You can use [editorconfig](http://editorconfig.org/) if you want to get the text editor to convert your `tab` keys as 4-spaces.